.evergreen/csfle/start_servers.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+# start the kmip server in the background.
+set -eu
+
+SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+. $SCRIPT_DIR/../handle-paths.sh
+pushd $SCRIPT_DIR
+
+if [ ! -f ./secrets-export.sh ]; then
+    echo "Please run the setup_secrets.sh script"
+    exit 1
+fi
+
+source ./secrets-export.sh
+
+if [ -z "${CSFLE_TLS_CA_FILE-}" ]; then
+    echo "Please run the setup_secrets.sh script"
+    exit 1
+fi
+
+. ./stop_servers.sh
+. ./activate-kmstlsvenv.sh
+
+# The -u options forces the stdout and stderr streams to be unbuffered.
+# TMPDIR is required to avoid "AF_UNIX path too long" errors.
+echo "Starting KMIP Server..."
+TMPDIR="$(dirname "$DRIVERS_TOOLS")" python -u kms_kmip_server.py --ca_file $CSFLE_TLS_CA_FILE --cert_file $CSFLE_TLS_CERT_FILE --port 5698 &
+echo "$!" > kmip_pids.pid
+echo "Starting KMIP Server...done."
+sleep 1
+
+echo "Starting HTTP Server 1..."
+python -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/expired.pem --port 8000 > http1.log 2>&1 &
```

Comment:
Suggest using ports to match the spec test readme: 9000, 9001, and 9002: https://github.com/mongodb/specifications/blob/cea087babea68571fa4068ace6b2d326c3a0a22d/source/client-side-encryption/tests/README.rst#L1385

Reply:
Done